packages/workbox-build/src/schema/GenerateSWOptions.json

```diff
@@ -857,7 +852,16 @@
     },
     "URLSearchParams": {
       "type": "object",
-      "additionalProperties": false
+      "properties": {
+        "size": {
+          "description": "The total number of parameter entries.",
+          "type": "number"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "size"
+      ]
```

Comment:
For this and all other changes in `packages/workbox-build/src/schema`, where do those changes come from? I assume they're auto generated, but I'd like to better understand it. Is there a script that triggers the updates, or does it automatically happen after a dependency update? Where is that set up? 🤔

Reply:
`npx gulp build_node_packages` and `npx gulp build_node_ts_packages`, see `gulp-tasks/build-node-packages.js`